build.sh

```diff
@@ -182,6 +190,8 @@ echo ""
 echo "Cleaning plugins directory from NPM artifacts..."
 
 rm -rf "plugins/exportpdf"
+rm -rf "plugins/scayt"
+rm -rf "plugins/wsc"
```

Comment:
I think we could completely remove `plugins` dir here and also remove it from repository itself - so it will be just temporary directory during script is executed.